packages/libs/chainweb-node-client/package.json

```diff
@@ -35,7 +35,7 @@
     "format:md": "remark README.md -o --use @kadena-dev/markdown",
     "format:src": "prettier config openapi src --write",
     "generate:openapi-types": "echo 'openapi specs needs fixes' # openapi-typescript \"./openapi/*.json\" --output ./src/openapi",
-    "postinstall": "pnpm run generate:openapi-types",
```

Comment:
Shouldn't this be part of the build script too?

```
"build": "pnpm run generate:openapi-types && heft build --clean",
```

Reply:
Didn't bother too much, this isn't doing anything anyway (just `echo`)